spec/services/step_nav_publisher_spec.rb

```diff
@@ -29,6 +31,40 @@
         expect(Services.publishing_api).to have_received(:put_content).with(content_id, payload)
       end
     end
+
+    context "marks draft updated" do
```

Comment:
Kevin has picked up on this in one of my PRs, but our use of `context` is quite confusing. I'd expect this to be a `describe "whether a step by step should be reverted to draft when edited"`, _or_ a `context "there is a step by step page with steps"`.

But it looks like `context` is used in this way throughout the file, so probably one to tackle in a future cleanup PR 🤔